### DISCUSSION TITLE
what does it do?

### DISCUSSION THREAD

**Discussion by mcerrick** [reupen/console_panel#140] [Q&A] [answered]
the description is VERY short. What would be the benefit of installing this?

**Comment by reupen** [comment, marked as answer]
It's a [foobar2000](https://www.foobar2000.org/) component. In foobar2000, there is a window called the Console which can be accessed from the View menu.

Console panel is for users of the [Columns UI component for foobar2000](https://yuo.be/columns-ui). It gives you a version of the console that can be embedded in the Columns UI main window, rather than as a pop-up window.
